### PR TITLE
Let serviceB return json

### PR DESCRIPTION
**Pull request by tevans78**

### CODE REVIEW

**Comment by jonhawkes** [comment]
This works fine but the documentation in https://github.com/eclipse/microprofile-service-mesh/pull/27 now needs updating to reflect the new config property.

File: src/main/java/org/eclipse/microprofile/servicemesh/serviceb/CallCounter.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import javax.enterprise.context.ApplicationScoped;
+
+/**
+ * A CDI ApplicationScoped wrapper around an AtomicInteger
+ */
+@ApplicationScoped
+public class CallCounter {
```
Comment (pilhuhn) [inline src/main/java/org/eclipse/microprofile/servicemesh/serviceb/CallCounter.java:31]: Why are you not using a counter from MP-Metrics?

File: src/main/java/org/eclipse/microprofile/servicemesh/serviceb/ServiceBEndpoint.java [hunk 2]
```diff
@@ -25,46 +25,29 @@
...
+        return data.getMessage() + " " + data.getSource();
     }
-
-    try {
-      hostname = InetAddress.getLocalHost().getHostName();
-    } catch(java.net.UnknownHostException e) {
-      hostname = e.getMessage();
+    
+    @GET
+    @Produces({MediaType.APPLICATION_JSON,})
```
Comment (pilhuhn) [inline src/main/java/org/eclipse/microprofile/servicemesh/serviceb/ServiceBEndpoint.java]: Extra `,` at the end

**Review by pilhuhn — CHANGES_REQUESTED** [review]

**Comment by pilhuhn** [comment]
Also the title of the PR is a bit confusing - especially as the output is now also json, but there is no explicit use of json-b.
And if there were explicit usage, then I would be worried, as not may servers support EE8 yet, which introduces json-b as API afair, but correct me if I am wrong.

File: src/main/java/org/eclipse/microprofile/servicemesh/serviceb/CallCounter.java (hunk 1 above)
Comment (tevans78) [reply src/main/java/org/eclipse/microprofile/servicemesh/serviceb/CallCounter.java:31]: Good idea, will do that in a different PR though

**Comment by tevans78** [comment]
Sorry, it was a typo ... I am using jsonp

File: src/main/java/org/eclipse/microprofile/servicemesh/serviceb/ServiceBEndpoint.java (hunk 2 above)
Comment (tevans78) [reply src/main/java/org/eclipse/microprofile/servicemesh/serviceb/ServiceBEndpoint.java]: I'll fix that now

**Comment by tevans78** [comment]
fixes #5

**Review by pilhuhn — APPROVED** [review]

**Comment by pilhuhn** [comment]
Approving given that changes for counter will be in #6

**Pull request merged**